README.md

```diff
@@ -1,79 +1,86 @@
+
 # BreezewayMod
+
 ## About
-![](breezewaymod_icon.png) 
 
-BreezewayMod is a fork of the Breezeway window decoration which aims to mimic the macOS interface. But with powerups.
-Did I mention that it comes without weird glitches on the rounded corners?
+<img src="breezewaymod_icon.png" style="zoom: 25%;" /> 
 
-## Screenshots
-![](screenshot.png)
+**BreezewayMod** is a fork of the [Breezeway](https://github.com/richard-kung/breezeway) window decoration, which aims to mimic the *macOS style*, but with **power ups**.
 
-Looks good with light and dark color schemes (shown here with color schemes "Arc" for light windows and "Arc-Dark" for dark windows)
+> **Note**: it comes without weird glitches on the rounded corners.
 
-![](sierrabreeze.png)
+## Preview
 
-The usual glitch on dark themes.
+![](screenshot.png)
 
-![](breezewaymod.png)
+> Looks great  with light and dark color schemes. Shown here with [Arc](https://github.com/PapirusDevelopmentTeam/arc-kde) and [Arc-Dark](https://github.com/PapirusDevelopmentTeam/arc-kde) color schemes.
 
-Glitch-b-gone with Breezeway magick!
 
-![](low_contrast.png)
+| The usual glitch on dark themes            | ![](sierrabreeze.png) |
+| ------------------------------------------ | :-------------------: |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](breezewaymod.png) |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](low_contrast.png) |
 
-If overlapping windows have not enough contrast between each other it's sometimes hard to see overlapping titlebars of you use a flat theme. Luckily, BreezewayMod comes with a neat highlight line across the top of your titlebar to get that sorted out!
+On overlapping windows with not enough contrast between each other, it's hard to see overlapping titlebars if using a flat theme. Luckily, **BreezewayMod** comes with a neat highlight line across the top of titlebars to get that sorted out:
```

Comment:
This would be a good text for line 22. Maybe shorten it tho.

Reply:
Done.